### PR TITLE
Add Project tab

### PR DESCRIPTION
**Pull request by SnowNooDLe**
# Project tab

Now we have a project tab that interacts with dropdowns.

image

Unlike the Hazard Analysis tab, users can choose from the dropdown we provided.

## Workflow

It is pretty much similar to Seismic Hazard.

image

To get the **Regional and VS30 maps**, do the following steps

1. Choose the `Project ID`
2. Choose the `Location`
3. Choose the `VS30`
4. Click the `Get` button

image

To get the **Hazard Curve**

1. Choose `Intensity Measure`
2. Click the `Get` button

To get the **Disaggergation**

1. Choose `Intensity Measure` first (No need to click Get button as we just need IM value)
2. Choose `Return Period` from the **Disaggregation** section
3. Click the `Get` button

To get the **UHS**

1. Choose `Return periods` from **Uniform Hazard Spectrum** (You can choose more than one but currently we only have one)
2. Click the `Get` button

### Regional Map
image

### VS30 Map
image

### Hazard Curve

image

image

### Disaggregation

image

image

image

### UHS

image

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -250,72 +299,158 @@ With a directory called `docker`, there are two more directories, `dev` and `ea`
...
+#### Project Seismic Hazard
 
-> env-cmd -f .env.production react-scripts build -- --node-flags --max-old-space-size=8192
+Users need to set the location first in **Site Selection** tab to get into this tab.
 
-Where "env-cmd" is a plugin which passes the file .env.production to the build, and "-- --node-flags --max-old-space-size=8192" increase the stack allocation.
+##### Project Hazard Curve
 
-I've found that I need to run these commands in my gitBash terminal, as they don't work reliable using the vsCode commandline.
+Users can choose one **Intensit Measure** from the dropdown, then click **Get** button to get the following plots.
```
Comment (jasonmotha) [inline frontend/README.md:439]: spelling

File: frontend/src/components/Project/SiteSelection/SiteSelectionForm.js [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+              setProjectUHSRPs(responseUHSRPData["rps"]);
+              // Reset dropdowns
+              setLocalLocation(null);
+              setLocalVS30(null);
+            })
+            .catch((error) => {
+              console.log(error);
+            });
+        } catch (error) {
+          console.log(error);
```
Comment (jasonmotha) [inline frontend/src/components/Project/SiteSelection/SiteSelectionForm.js:158]: Do these errors need the error modal?

**Review by jasonmotha — CHANGES_REQUESTED** [review]

File: frontend/README.md (hunk 1 above)
Comment (SnowNooDLe) [reply frontend/README.md]: Good catch, I went through Grammarly and though was okay.

File: frontend/src/components/Project/SiteSelection/SiteSelectionForm.js (hunk 2 above)
Comment (SnowNooDLe) [reply frontend/src/components/Project/SiteSelection/SiteSelectionForm.js:158]: In regard to 'Error Modal', is this what you mean? 
Screenshot from 2020-11-27 14-21-16

If so, I think things like these can be done when we add Tooltips ? as if it was for Viewer part (ex: GMS Viewer, Hazard Curve viewer) then they have a dedicated space to display a warning msg.

Whereas this part (sending a request to Project API to get data in JSON) and maybe getting IMs after we set location in Site Selection for Hazard Analysis, they only have a dropdown and disabled as we failed to fetch so the length of the option is 0 (As the option is an array for react-select)

**Review by jasonmotha — APPROVED** [review]

File: frontend/README.md [hunk 3]
```diff
@@ -250,72 +299,158 @@ With a directory called `docker`, there are two more directories, `dev` and `ea`
...
+## Application
+
+### Hazard Analysis
+
+#### Site Selection
+
+##### Ensemble
+
+If it is in DEV version, you see two options, **v20p5emp** and **v20p45sim**.
+But in EA and PROD.
```
Comment (claudio525) [inline frontend/README.md:311]: This sentence seems incomplete?

**Review by claudio525 — APPROVED** [review]

**Pull request merged**